manifests/proxydhcp.pp

```diff
@@ -49,6 +49,12 @@
     failover       => $failover,
   }
 
+  exec { 'setfacl_dhcp':
+    command => 'setfacl -R -m u:theforeman:rwx /etc/dhcp /var/lib/dhcpd',
+    path => '/usr/bin',
+    onlyif => 'test -f /var/lib/dhcp && test -f /etc/dhcp',
```

Comment:
> I am not familiar with what extended attributes are.

Linux stores the ACLs (and things like SELinux attributes) in extended attributes. It is possible to compile a kernel or have an FS without extended attributes. I'm not 100% sure but in some cases NFS might not support it for example.